go/store/datas/database_common.go

```diff
@@ -547,6 +557,16 @@ func (db *database) doUpdateWorkingSet(ctx context.Context, datasetID string, wo
 		return err
 	}
 
+	workingSetRef, err := db.WriteValue(ctx, workingSet) // will be orphaned if the tryCommitChunks() below fails
+	if err != nil {
+		return err
+	}
+
+	wsValRef, err := types.ToRefOfValue(workingSetRef, db.Format())
+	if err != nil {
+		return err
+	}
+
 	// This could loop forever, given enough simultaneous writers. BUG 2565
```

Comment:
Which issue is 2565? Not seeing that as a valid issue number

Reply:
It was an issue in the noms repo.

I'm just deleting the comment